### PR TITLE
Make http proxy a bidi stream

### PR DESCRIPTION
**Pull request by tjholm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "repository": "https://github.com/nitrictech/node-sdk",
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "version": "0.0.1",
```
Comment (davemooreuws) [inline package.json:9]: remove

File: src/resources/http.ts [hunk 2]
```diff
@@ -16,12 +16,12 @@ import portfinder from 'portfinder';
 import { HttpClient } from '@nitric/proto/http/v1/http_grpc_pb';
 import { SERVICE_BIND } from '../constants';
 import * as grpc from '@grpc/grpc-js';
-import { HttpProxyRequest } from '@nitric/proto/http/v1/http_pb';
-import { HttpContext } from '../context/http';
+import { ClientMessage, HttpProxyRequest } from '@nitric/proto/http/v1/http_pb';
+import * as nodeHttp from 'http';
```
Comment (davemooreuws) [inline src/resources/http.ts:20]: Can we import the Server type from this instead?

File: package.json (hunk 1 above)
Comment (tjholm) [reply package.json]: done

File: src/resources/http.ts (hunk 2 above)
Comment (tjholm) [reply src/resources/http.ts]: done

**Review by jyecusch — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**